reloc.ml

```diff
@@ -68,7 +68,7 @@ let read_file fn =
 let get_output ?(use_bash = false) ?(accept_error=false) cmd =
   let fn = Filename.temp_file "flexdll" "" in
   let cmd' = cmd ^ " > " ^ (Filename.quote fn) in
-    if String.length cmd' < 8182 && not use_bash then
+    if String.length cmd' < 8161 && not use_bash then
```

Comment:
I would suggest defining this above `get_output`:

```ocaml
(* This is the longest command which can be passed to `Sys.command` *)
let max_native_command_length =
  let processor = try Sys.getenv "COMSPEC" with Not_found -> "cmd.exe" in
  (* The 4 is from the " /c " *)
  8191 - String.length processor - 4
```
and then:
```suggestion
    if String.length cmd' < max_command_length && not use_bash then
```

Reply:
Done!